utils/paste.tsx

```diff
@@ -71,13 +87,30 @@ export function formatMarkdownTableMessage(table: HTMLTableElement, message?: st
     }).join('\n');
 
     const formattedTable = `${header}${body}\n`;
+    return formattedTable;
```

Comment:
nit and no mistake from you, but this variable is unnecessary:

```suggestion
    return `${header}${body}\n`;
```